app/services/check_availability_task_service.rb

```diff
@@ -14,6 +14,24 @@ def process
 
   private
 
+  def refresh
+    endpoints = Sources::Service.call do |api_instance|
+      api_instance.list_endpoints(:filter => {:source_id => source_id})
+    end
+    endpoint = endpoints.try(:data).try(:first)
+
+    applications = Sources::Service.call do |api_instance|
+      api_instance.list_applications(:filter => {
+                                       :source_id           => source_id,
+                                       :application_type_id => ClowderConfig.instance["APPLICATION_TYPE_ID"]
```

Comment:
@hsong-rh  Is the application_type_id an int or a string here?

Reply:
both `source_type_id` and `application_type_id` are int
https://github.com/RedHatInsights/catalog_inventory-api/blob/master/config/initializers/init_types_envs.rb#L18

Reply:
@hsong-rh But is the sources API expecting it to be strings or int from their openapi.json?

Reply:
sources API client honor both string and int